ts/IterableLogger.ts

```diff
@@ -0,0 +1,15 @@
+import { Iterable } from "./Iterable"
+
+class IterableLogger {
+    log(message: String) {
+        // default to `true` in the case of unit testing where `Iterable` is not initialized
+        // which is most likely in a debug environment anyways
+        var loggingEnabled = Iterable.savedConfig?.logReactNativeSdkCalls ?? true
+
+        if (loggingEnabled) {
```

Comment:
Should we not have different levels on RN side as well? Like INFO, Debug , Error?

Reply:
We can support that in a future version, but for now, we don't exactly have that distinction either, since the RN SDK doesn't pass along errors from the native layer, either.